docs/intro.txt

```diff
@@ -10,9 +10,9 @@ to be simulated using ``stim``.
 ``pytket-stim`` is available for Python 3.9, 3.10 and 3.11, on Linux, MacOS and
 Windows. To install, run:
 
-::
-   
-   pip install pytket-stim
+```shell
```

Comment:
Ah there is no need to change this in the index as the install command is already centered there

https://tket.quantinuum.com/extensions/pytket-stim/

Its just the mardown syntax in the README that needs updated.